src/Serilog.Sinks.MongoDB/project.json

```diff
@@ -0,0 +1,21 @@
+﻿{
+  "version": "2.0.0-beta-*",
+  "description": "The MongoDB sink for Serilog",
+  "authors": [ "Kiran Makkapati", "Serilog Contributors" ],
+  "tags": [ "serilog", "mongodb" ],
+  "projectUrl": "http://serilog.net",
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "iconUrl": "http://serilog.net/images/serilog-sink-nuget.png",
+  "dependencies": {
+    "Serilog": "2.0.0-beta-511",
+    "Serilog.Sinks.PeriodicBatching": "2.0.0-beta-700"
+  },
+
+  "frameworks": {
+    "dnx46": {
```

Comment:
I don't know what the plan is here, but we shouldn't ship anything with `dnx46` - it's an app-only TFM. Libs should ship with `net46` or preferably `dotnet5.4`/`netstandard1.3`.